frontend/src/App.tsx

```
@@ -1,5 +1,46 @@
+import { createRouter, RouterProvider } from "@tanstack/react-router";
+
+import AuthProvider from "@/context/AuthProvider";
+import ThemeProvider from "@/context/ThemeProvider";
+import { useAuth } from "@/context/use-auth";
+import NotFoundErrorPage from "@/pages/NotFoundErrorPage";
+import { routeTree } from "@/routeTree.gen";
+
+const router = createRouter({
+  routeTree,
+  context: {
+    auth: undefined!, // This will be set after we wrap the app in an AuthProvider
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)